### DISCUSSION TITLE
Versions of Tonic

### DISCUSSION THREAD

**Discussion by mountains-high** [neuromorphs/tonic#106] [Q&A] [answered]
Good day,

I want to work on older versions of Tonic. Could you tell me please, from where I can install them?
Thank you!

**Comment by biphasic** [comment, marked as answer]
you can always install older versions by pinning them, for example `pip install tonic==0.3.4`
see the full list of releases here: https://pypi.org/project/tonic/#history
alternatively you can check out the repository, change to a specific commit, and then run `pip install .` this will install the local version of your liking

**Reply by mountains-high** [reply]
Thank you
